wiki/Community/Mappers_Guild/id.md

```diff
@@ -56,7 +56,11 @@ Para anggota Mappers' Guild juga dapat memperoleh poin tambahan dengan mendaftar
 
 *Artikel utama: [Kumpulan beatmap-beatmap showcase Featured Artist](/wiki/Featured_Artists/Featured_Artist_showcase_beatmaps)*
 
-Anggota-anggota Mappers' Guild tertentu terkadang diberikan kesempatan untuk membuat "beatmap-beatmap showcase" eksklusif yang diperuntukkan bagi pengumuman-pengumuman Featured Artist di masa mendatang. Setelah Featured Artist yang bersangkutan diumumkan, beatmap-beatmap tersebut akan diberikan sorotan khusus pada [postingan berita yang terkait](https://osu.ppy.sh/home/news) dan sering kali juga dipromosikan di Twitter [@osugame](https://twitter.com/osugame). 
+Anggota-anggota Mappers' Guild tertentu terkadang diberikan kesempatan untuk membuat "beatmap-beatmap showcase" eksklusif yang diperuntukkan bagi pengumuman-pengumuman Featured Artist di masa mendatang. Setelah Featured Artist yang bersangkutan diumumkan, beatmap-beatmap tersebut akan diberikan sorotan khusus pada [postingan berita yang terkait](https://osu.ppy.sh/home/news) dan sering kali juga dipromosikan di Twitter [@osugame](https://twitter.com/osugame).
+
+### Kontes Beatmapping
+
+Mappers' Guild juga digunakan untuk mengelola kontes beatmapping. Sejak Januari 2020, semua perulangan dari [Kontes Beatmapping Bulanan](/wiki/Contests/Monthly_Beatmapping_Contest) telah di-host, disaring, dan dinilai menggunakan sistem yang disediakan oleh Mappers' Guild. Setiap pengguna dapat menge-host kontes melalui Mappers' Guild untuk meningkatkan aksesibilitas ke komunitas mapping osu!.
```

Comment:
```suggestion
Mappers' Guild juga berfungsi selaku sarana kontes beatmapping. Sejak Januari 2020, seluruh iterasi [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) telah diselenggarakan, dijurikan, dan dinilai dengan sistem yang ditentukan oleh Mappers' Guild. Di samping itu, siapapun dapat menyelenggarakan kontesnya tersendiri melalui Mappers' Guild untuk meningkatkan aksesibilitas ke komunitas mapping.
```

Reply:
not sure if `Monthly Beatmapping Contests` should be translated tbh, as -- just like e.g. `A Labour of Love` or `Pending Cup` or so on -- "Monthly Beatmapping Contests" is literally the name of the event